File: .ProgramingBasics/.SimpleConditionalStatements/14.AreaOfFigures/src/AreaOfFigures.java
import java.util.Scanner;

public class AreaOfFigures {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        double area;

        switch (input) {
            case "square": {
                double side = Double.parseDouble(scanner.nextLine());
                area = Math.pow(side, 2);
                System.out.println(area);
                break;
            }
            case "rectangle":
                double sideA = Double.parseDouble(scanner.nextLine());
                double sideB = Double.parseDouble(scanner.nextLine());
                area = sideA * sideB;
                System.out.println(area);
                break;
            case "circle":
                double radius = Double.parseDouble(scanner.nextLine());
                area = Math.PI * Math.pow(radius, 2);
                System.out.println(area);
                break;
            case "triangle": {
                double side = Double.parseDouble(scanner.nextLine());
                double height = Double.parseDouble(scanner.nextLine());
                area = (side * height) / 2.0;
                System.out.println(area);
                break;
            }
        }
    }
}

File: ProgramingBasics/SimpleConditionalStatements/13.AreaOfFigures/13.AreaOfFigures/AreaOfFigures.cs
using System;

namespace _13.AreaOfFigures
{
    public class AreaOfFigures
    {
        public static void Main(string[] args)
        {
            var figure = Console.ReadLine().ToLower();
            var areaOfFigure = 0.0;

            switch (figure)
            {
                case "square":
                    var side = double.Parse(Console.ReadLine());
                    areaOfFigure = side * side;
                    break;
                case "rectangle":
                    var a = double.Parse(Console.ReadLine());
                    var b = double.Parse(Console.ReadLine());
                    areaOfFigure = a * b;
                    break;
                case "circle":
                    var radius = double.Parse(Console.ReadLine());
                    areaOfFigure = Math.PI * radius * radius;
                    break;
                case "triangle":
                    var sideT = double.Parse(Console.ReadLine());
                    var h = double.Parse(Console.ReadLine());
                    areaOfFigure = (sideT * h) / 2;
                    break;
            }

            Console.WriteLine(Math.Round(areaOfFigure, 3));
        }
    }
}